lib/view_component/engine.rb

```diff
@@ -87,9 +88,11 @@ class Engine < Rails::Engine # :nodoc:
       options = app.config.view_component
 
       if options.show_previews
+        preview_controller = options.preview_controller.sub(/Controller$/, "").underscore
+
         app.routes.prepend do
-          get options.preview_route, to: "view_components#index", as: :preview_view_components, internal: true
-          get "#{options.preview_route}/*path", to: "view_components#previews", as: :preview_view_component, internal: true
+          get options.preview_route, to: "#{preview_controller}#index", as: :preview_view_components, internal: true
+          get "#{options.preview_route}/*path", to: "#{preview_controller}#previews", as: :preview_view_component, internal: true
```

Comment:
Shouldn't these two use the underscored version of the controller name? (and without the "Controller" substring)

Reply:
Yes, they are -- see above.

Reply:
Nice one!